lib/gs-app-private.h

```diff
@@ -24,6 +24,11 @@
 
 #include "gs-app.h"
 
+typedef enum {
+	GS_APP_FLAG_NONE			= 0,
+	GS_APP_FLAG_PENDING_STATE_CHANGE	= 1 << 0
```

Comment:
That's exactly what I first wrote and proposed to Richard but he said we wanted to have the flags instead. I think it's because we'll be migrating some quirks out of `appstream-glib` to `GsApp`. This is also private API, so we can change it whenever we want.